tox.ini

```diff
@@ -16,6 +16,13 @@ deps =
     xgboostlatest: xgboost
     -r{toxinidir}/requirements.txt
     -r{toxinidir}/test-requirements.txt
+conda_deps=
```

Comment:
can you comment on why these were added?

Reply:
I was only able to run the tests successfully by specifying these dependency versions. Otherwise the versions installed by conda lead to errors during test run, for example `mlio` not being able to find `libtbb` file because the file had a different suffix for the version.
I wanted to see if others were able to run the unit tests locally, without any issues. I can drop this out of the CR if noone else had any issues.

Reply:
got it, I think its fine, just makes things more robust to building on different setups.